pinot-common/src/main/java/org/apache/pinot/common/exception/QueryException.java

```diff
@@ -54,7 +54,7 @@ public static void setMaxLinesOfStackTrace(int maxLinesOfStackTracePerFrame) {
   public static final int ACCESS_DENIED_ERROR_CODE = 180;
   public static final int TABLE_DOES_NOT_EXIST_ERROR_CODE = 190;
   public static final int QUERY_EXECUTION_ERROR_CODE = 200;
-  public static final int QUERY_CANCELLATION_ERROR_CODE = 205;
+  public static final int QUERY_CANCELLATION_ERROR_CODE = 503;
```

Comment:
is it possible to add a unit test say similar to `CombineErrorOperatorsTest`?

Reply:
As the error is returned from the runner thread not worker threads through the result block, having the error code test in OfflineClusterMemBasedServerQueryKilingTest  and OfflineClusterMemBasedBrokerQueryKilingTest and  would suffice

Reply:
sounds good. it case should be good enough